src/main/java/Strings/PleaseDeleteMe.java

```diff
@@ -0,0 +1,53 @@
+package Strings;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PleaseDeleteMe {
+
+    String name ="ray maldoNADo ESTrada";
+
+    @Test
+    void ShouldReturnVowelsOnly(){
+
+        List<Character> vowels = new ArrayList<>();
+        vowels.add('A');
+        vowels.add('E');
+        vowels.add('I');
+        vowels.add('O');
+        vowels.add('U');
+
+        char[] nameChar = name.toCharArray();
+
+
+
+                 int vowelsCount = 0;
+        int consonantsCount = 0;
+                 int emptySpaces = 0;
+
+        for(int i = 0; i <= nameChar.length - 1; i++){
+            if (vowels.contains(Character.toUpperCase(nameChar[i]))) {
```

Comment:
To make this more readeable, assuming we want to pass a normalized string in the if statement. Could we use a general function / method to normalize the strings to be passed in then do the comparison or call it wherever is needed ?

Reply:
@rayme11 please see comments above